src/cljdoc/util/sqlite_cache.clj

```diff
@@ -13,11 +13,21 @@
             [clojure.java.jdbc :as sql])
   (:import (java.time Instant)))
 
+(defn instant-now []
+  (Instant/now))
+
+(defn epoch-time-milliseconds []
+  (.toEpochMilli (instant-now)))
```

Comment:
I personally don't see a lot of value in these kinds of aliases. IMHO they add a layer of indirection that doesn't really add much value. I don't mind keeping this in but just wanted to share a, perhaps differing, viewpoint.

Reply:
Oh, did not notice this comment until now.
`instant-now` is for test support so we can redef time easily. I will add a comment.
`epoch-time-milliseconds` was not used for test support - I can turf.